api/grpcserver/grpc.go

```diff
@@ -19,28 +19,31 @@ type ServiceAPI interface {
 // Server is a very basic grpc server.
 type Server struct {
 	Listener   string
+	logger     *zap.Logger
```

Comment:
Why be inconsistent here? `log.Logger` is used everywhere else.

Reply:
Because of line 62 where a suggared logger is used. If that log can be structured as well then the `log.Logger` can be used.

Reply:
`Logger::Info()` works identically as zap's sugared logger. In fact, `log.Log` uses it underneath: https://github.com/spacemeshos/go-spacemesh/blob/c4275d506b32a71dbb40ab485b1e65554db0ef6f/log/zap.go#L25-L27

Reply:
Updated